projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -608,6 +608,112 @@ function wpcom_register_launchpad_task_list( $task_list ) {
 	return wpcom_launchpad_checklists()->register_task_list( $task_list );
 }
 
+/**
+ * Retrieves the required tasks from a given set of tasks.
+ *
+ * This function filters the provided tasks array and returns only the tasks that are marked as required, based on the specified
+ * array of required task IDs.
```

Comment:
Nit: It may be worth introducing the newline earlier.
```suggestion
 * This function filters the provided tasks array and returns only the tasks that are marked as required,
 * based on the specified array of required task IDs.
```

Reply:
It may also be worth noting that some tasks may not be visible to the user, so we will only return required tasks that are in the task list, meaning we may be "missing" some tasks.